scripts/build.ts

```diff
@@ -14,8 +14,14 @@ import { sysNode, sysNodeExternalBundles } from './bundles/sys-node';
 import { testing } from './bundles/testing';
 import { validateBuild } from './test/validate-build';
 import { rollup } from 'rollup';
+import type { RollupOptions } from 'rollup';
 
-export async function run(rootDir: string, args: string[]) {
+/**
+ * Runner for releasing a new version of Stencil
+ * @param rootDir the root directory of the Stencil repository
+ * @param args stringifed arguments that influence the release process
+ */
+export async function run(rootDir: string, args: ReadonlyArray<string>): Promise<void> {
```

Comment:
Generally, if I could use the readonly variant of arrays, I did. Arguments and the like probably shouldn't change while we're doing a release

Reply:
TIL!